src/test/java/uk/gov/hmcts/darts/common/util/CommonTestDataUtil.java

```diff
@@ -427,4 +441,36 @@ public TranscriptionUrgencyEntity createTranscriptionUrgencyEntityFromEnum(Trans
         return transcriptionUrgencyEntity;
     }
 
+    public List<UserAccountEntity> createUserAccountEntities(int accounts, String emailAddress) {
+        List<UserAccountEntity> userAccountEntities = new ArrayList<>();
+        int id = 1;
+        for (int counter = 1; counter <= accounts; counter++) {
+            userAccountEntities.add(createUserAccount(id, emailAddress));
+            id++;
+        }
+
+        return userAccountEntities;
+    }
```

Comment:
Is the purpose of this method to create a specified number of user accounts with the same email address? If so, what's that for as I can only see `accounts` being passed in as `1` which would create a single user account?

Reply:
As there will always be only ever one result, this has been removed